docs/model/crdt-documents.md

```diff
@@ -181,14 +181,15 @@ An image element
 
 #### Fields
 
-| Field      | Type       | Description                                                                                               |
-| ---------- | ---------- | --------------------------------------------------------------------------------------------------------- |
-| `type`     | `'image'`  | Identifies the element as an image.                                                                       |
-| `mxc`      | `'string'` | [MXC URI](https://spec.matrix.org/v1.9/client-server-api/#matrix-content-mxc-uris) pointing to the image. |
-| `fileName` | `string`   | Image file name.                                                                                          |
-| `position` | `Point`    | The position of the image on the whiteboard canvas.                                                       |
-| `width`    | `number`   | Scaling of the image on the x-axis.                                                                       |
-| `height`   | `number`   | Scaling of the image on the y-axis.                                                                       |
+| Field       | Type       | Description                                                                                               |
+| ----------- | ---------- | --------------------------------------------------------------------------------------------------------- |
+| `type`      | `'image'`  | Identifies the element as an image.                                                                       |
+| `mxc`       | `'string'` | [MXC URI](https://spec.matrix.org/v1.9/client-server-api/#matrix-content-mxc-uris) pointing to the image. |
+| `fileName`  | `string`   | Image file name.                                                                                          |
+| `position`  | `Point`    | The position of the image on the whiteboard canvas.                                                       |
+| `width`     | `number`   | Scaling of the image on the x-axis.                                                                       |
+| `height`    | `number`   | Scaling of the image on the y-axis.                                                                       |
+| `available` | `bool`     | False if the image URL is not available for retrieval                                                     |
```

Comment:
I agree, we should not put image loading status into the model.

We could put it to a react context, a separate one sounds good to me.

Or we can not care that placeholder can be copied. I think disabling copy has his own cons in case of multiselect.

Reply:
I've reverted the hidding of the duplicate option on the placeholder - we will handle that on another ticket, as agreed earlier today.